<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Consolas</family>
       </font>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_connect"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_4"/>
   <addaction name="action"/>
   <addaction name="actionRefresh"/>
   <addaction name="actionNewfolder"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_connect">
   <property name="text">
    <string>连接(connect)</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/Upload.png</normaloff>:/images/images/Upload.png</iconset>
   </property>
   <property name="text">
    <string>上传</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/connect.png</normaloff>:/images/images/connect.png</iconset>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
   <property name="toolTip">
    <string>connect</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/Refresh.png</normaloff>:/images/images/Refresh.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actionNewfolder">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/images/NewFolder.png</normaloff>:/images/images/NewFolder.png</iconset>
   </property>
   <property name="text">
    <string>newfolder</string>
   </property>
   <property name="toolTip">
    <string>add folder</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_4</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onLogin()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onUpload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRefresh</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>onRefreshWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onLogin()</slot>
  <slot>onBack()</slot>
  <slot>onUpload()</slot>
  <slot>onRefreshWindow()</slot>
 </slots>
</ui>
